### PR TITLE
gstreamer family 1.12.3

### PR DESCRIPTION
**Pull request by ilovezfs**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

File: Formula/gst-plugins-ugly.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class GstPluginsUgly < Formula
   desc "Library for constructing graphs of media-handling components"
   homepage "https://gstreamer.freedesktop.org/"
-  url "https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.12.2.tar.xz"
-  sha256 "1cc3942bbf3ea87da3e35437d4e014e991b103db22a6174f62a98c89c3f5f466"
+  url "https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.12.3.tar.xz"
+  sha256 "e88ca584c94ea78eeecbf3af00ef7f134b66bdee7408aa4aa6c547235e060052"
   revision 1
```
Comment (philn) [inline Formula/gst-plugins-ugly.rb]: Drop this?

File: Formula/gst-plugins-bad.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class GstPluginsBad < Formula
   desc "GStreamer plugins less supported, not fully tested"
   homepage "https://gstreamer.freedesktop.org/"
-  url "https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.12.2.tar.xz"
-  sha256 "9c2c7edde4f59d74eb414e0701c55131f562e5c605a3ce9b091754f106c09e37"
+  url "https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.12.3.tar.xz"
+  sha256 "36d059761852bed0f1a7fcd3ef64a8aeecab95d2bca53cd6aa0f08054b1cbfec"
   revision 1
```
Comment (philn) [inline Formula/gst-plugins-bad.rb]: Ditto

**Comment by ilovezfs** [comment]
thx :)

**Comment by philn** [comment]
The CI fails for gst-python and this is exactly the issue I fixed with my pygobject3 commit :)
```
checking for PYGOBJECT... configure: error: Package requirements (pygobject-3.0 >= 3.0) were not met:

No package 'pygobject-3.0' found
```

**Comment by ilovezfs** [comment]
patience

**Comment by fraca7** [comment]
@philn Which commit ? I was about to create a pull request to fix the pygobject problem, do I spare myself the effort ? TIA.

**Comment by fraca7** [comment]
Very cool, thanks

**Pull request closed without merging**